src/Entity/Message.php

```diff
@@ -409,4 +410,16 @@ public function setLanguage($language) {
     $this->language = $language;
   }
 
+  /**
+   * Default value callback for 'uid' base field definition.
+   *
+   * @see ::baseFieldDefinitions()
```

Comment:
Perhaps this part of the comment re: baseFieldDefinitions should be updated, or even removed? If updated, maybe a comment to state it is used in modules such as Message UI?